### PR TITLE
Efrs 1241 image count in manage collections

### PR DESCRIPTION
**Pull request by smchedlidze826**

### CODE REVIEW

File: dev/docker-compose.yml [hunk 1]
```diff
@@ -80,12 +80,10 @@ services:
       - compreface-admin
 
   compreface-core:
-    image: ${registry}compreface-core:${CORE_VERSION}
+    image: pospielov/compreface-core:1.0.0-mobilenet-2d106
```
Comment (isobolev) [inline dev/docker-compose.yml:83]: is this change needed to be committed?

Comment (smchedlidze826) [reply dev/docker-compose.yml]: removed

**Review by isobolev — APPROVED** [review]
